### PR TITLE
#968 removing strange enters and spaces in formatting.

### PR DESCRIPTION
**Pull request by sjaakd**
Should be an easy one to review :smile:

### CODE REVIEW

File: processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl [hunk 1]
```diff
@@ -43,20 +43,16 @@
 -->
 <#macro callTargetWriteAccessor>
     <@lib.handleNullCheck>
-      <#if directAssignment>
-        ${ext.targetBeanName}.${ext.targetWriteAccessorName}( <@wrapLocalVarInCollectionInitializer/> );
-    <#else>
-        ${ext.targetBeanName}.${ext.targetWriteAccessorName}( ${localVarName} );
-    </#if>
```
Comment (filiphr) [inline processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl]: The compress single_line does not work for this one?

Comment (sjaakd) [inline processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl]: probably it does.. only In this case the whole thing fits on one line.. After all its the same call, only the arg differs.

Comment (filiphr) [inline processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl]: I agree that it fits. What I meant is that it is easier to read it and understand when you look at the template only

Comment (sjaakd) [inline processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl]: Hey @filiphr ,

I tried your suggestion.. But there's a side effect. The compress also acts on the call site. So the result is (see Issue289 for instance):

```java
        Collection<TargetElement> collection = sourceElementCollectionToTargetElementCollection( source.getCollection() ); if ( collection != null ) { targetWithSetter.setCollection( collection ); }        
```
In one line. Perhaps an error in Freemarker..

Comment (sjaakd) [inline processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl]: @agudian : this is spotted by Checkstyle on the generated code. However, the current situation (leading up to this Issue / PR) isn't. Can we catch the current situation as well?

Comment (sjaakd) [inline processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl]: I added another issue #969 to detect generated code formatting problems.

**Pull request merged**